instances/bodySystem/digestiveSystem.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/digestiveSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": "The 'digestive system' is an anatomical organ system is composed of organs devoted to the ingestion, digestion, the assimilation of food and the discharge of residual wastes.",
```

Comment:
"The 'digestive system' is an anatomical organ system composed..." (not "system IS composed")